src/Card.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './card.css';
+
+function Card (props){ 
+    const [ email, setEmail ] = React.useState('Loading...');
```

Comment:
The more I think about this, using a loading message is a clever way to deal with displaying a loading state. If you ever had a value where it didn't make sense to have a default, you could always use an `isLoading` boolean piece of state, and then conditionally do an early return if `isLoading` is true. This way is kind of nice because you can avoid that early return statement.